test-lib.sh

```diff
@@ -50,6 +50,33 @@ function ct_enable_cleanup() {
   trap ct_cleanup EXIT SIGINT
 }
 
+function ct_check_envs_set {
+  local env_filter check_envs env_format
+  env_filter=$1; shift
+  check_envs=$1; shift
+  loop_envs=$1; shift
+  env_format=${1:-"*VALUE*"}
+  while read -r variable; do
```

Comment:
Please add there a check, that all local variables has to be set.
For case, that somebody will call it directly.

Reply:
I can add something, but I think we are missing such a check for most of the library calls. I guess the expectation is that the caller should make sure its passed correctly?

Reply:
ok, let's leave and we will see in the future.